wall_e/extensions/ban.py

```diff
@@ -279,8 +291,13 @@ async def ban(self, ctx, user: discord.Member, *args):
                 await ctx.send(embed=e_obj)
             return
 
+        self.ban_progress.append(user.id)
```

Comment:
i can just remove this, i already add a unique constraint in the db to prevent duplicates. this was just to save some work like intercept wont fire while ban is working its thing.

Reply:
or maybe `ban_progress` should be a set instead since as far as I can tell, it will only ever contain a primitive type

Reply:
> so the db constraint will result in the intercept just silently failing?

yes